src/main/webapp/cdn/lib/xterm.js

```diff
@@ -1425,21 +1428,24 @@
                         case '\x10':
                             // PARALLAX: ASCII 10: clear
```

Comment:
The comment here is misleading, that's not ASCII 10, it's ASCII 16, or ASCII 0x10.